src/common/tvgMath.cpp

```diff
@@ -21,6 +21,7 @@
  */
 
 #include "tvgMath.h"
+#include <cmath>
```

Comment:
`tvgMath.h` also includes math. probably we can include one of them in `tvgMath.h`

Reply:
changed, on mac worked either way, waiting for android